### ISSUE TITLE
Banning and Unbanning from headless console doesn't persist after restart

### ISSUE THREAD

**Issue by Vegasx** [Neos-Metaverse/NeosPublic#2714]
### Describe the bug?

Reported by BigRedWolfy:

After a user is banned or unbanned from the console, restarting the
headless seems to undo any bans or unbans. So far the only way I've seen
for a ban to say registered is to log into the normal neos client with the
headless account and unban or ban from all my hosted sessions.

### Relevant issues

N/A

### To Reproduce

This has been tested with two headlesses, in each case I have tried to
unban Zagroseckt and ban the other headless (BigGreenWolfy and
SmallBlueFox). Both are running on the same pterodactyl panel on two
different servers with similar hardware. The pterydactyl panel is running
version 1.4.2 and both headlesses use an unofficial egg created by Synk
which was posted in the patreon channel a couple months ago. Both systems
are also running Ubuntu 20.04.2 Linux.

### Expected behavior

Users banned on a headless client should remain banned after restarting the headless client.

### Log Files

I've included logs and copied console text because commands typed into the console don't appear to show in the logs, at least in this case.

[BigGreenWolfy - Log 02 - Copied Console.log](https://github.com/Neos-Metaverse/NeosPublic/files/6881071/BigGreenWolfy.-.Log.02.-.Copied.Console.log)
[BigGreenWolfy - Log 01 - Raw.log](https://github.com/Neos-Metaverse/NeosPublic/files/6881072/BigGreenWolfy.-.Log.01.-.Raw.log)
[SmallBlueFox - Log 02 - Raw.log](https://github.com/Neos-Metaverse/NeosPublic/files/6881073/SmallBlueFox.-.Log.02.-.Raw.log)
[SmallBlueFox - Log 01 - Raw.log](https://github.com/Neos-Metaverse/NeosPublic/files/6881074/SmallBlueFox.-.Log.01.-.Raw.log)
[BigGreenWolfy - Log 02 - Raw.log](https://github.com/Neos-Metaverse/NeosPublic/files/6881075/BigGreenWolfy.-.Log.02.-.Raw.log)
[BigGreenWolfy - Log 01 - Copied Console.log](https://github.com/Neos-Metaverse/NeosPublic/files/6881076/BigGreenWolfy.-.Log.01.-.Copied.Console.log)
[SmallBlueFox - Log 02 - Copied Console.log](https://github.com/Neos-Metaverse/NeosPublic/files/6881077/SmallBlueFox.-.Log.02.-.Copied.Console.log)
[SmallBlueFox - Log 01 - Copied Console.log](https://github.com/Neos-Metaverse/NeosPublic/files/6881078/SmallBlueFox.-.Log.01.-.Copied.Console.log)

### How often does it happen?

Always

### Does the bug persist after restarting Neos?

Yes

### Neos Version Number

Beta 2021.7.25.489

### What Platforms does this occur on?

Windows

### Did this work before?

I Don't Know

**Comment by Frooxius** [comment]
I can't replicate this on my end, the ban persists correctly.

- Which commands are you using to ban the user?
- Is the ban listed when you use the listbans command?
- How exactly are you shutting down your headless?

**Comment by Medravr** [comment]
I think I can replicate this issue. Would you like me to add you to the Creator Jam group. I have working examples to use for testing.

**Comment by Medravr** [comment]
I have used the ban user command in the both the contacts list and the session menu. And unbanned the same user in the contacts list and debug menu.

The ban is listed when you use the listbans command--even after the unban has been created (note the world has been saved with the unban).

The headless is shutdown the proper way through commands.
Close (wait)
Shutdown.

**Comment by Frooxius** [comment]
@Medravr Are you using the Ban commands in Contacts list on the same account that's the headless? Or different account?

The bans aren't part of the world data itself, they're stored separately, so saving the world shouldn't make a difference there.

**Comment by Medravr** [comment]
Yes. I was using the ban commands in the same account as the headless. The ban however persists in separate group accounts too. Perhaps this is related to groups.
